designsafe/static/scripts/nco/components/nco-ttc-grants/nco-ttc-grants.template.html

```diff
@@ -3,19 +3,25 @@
     <div>Please make your selections and click the Search button to change the listing.</div>
     <div class="ttc-search">
         <form class="filter-group ttc-facility-filter">
-            <label for="ttc-facility-options">Filter by Facility:</label>
+            <label for="ttc-facility-options">Filter by Facility/Group:</label>
             <select name="ttc-facility-options" ng-model="$ctrl.selectedFacility">
-                <option value="">--Filter by Facility--</option>
+                <option value="">All</option>
                 <option ng-repeat="facility in $ctrl.facilitiesList" value="{{facility}}">{{facility}}</option>
             </select>
         </form>
         <form class="filter-group ttc-sort-filter">
             <label for="ttc-sort-options">Sort by:</label>
             <select name="ttc-sort-options" ng-model="$ctrl.selectedSort">
-                <option value="">--Sort Options--</option>
                 <option ng-repeat="sort in $ctrl.sortOptions" value="{{sort}}">{{sort}}</option>
             </select>
         </form>
+        <form class="filter-group ttc-category-filter">
+            <label for="ttc-category-options">Select a Category:</label>
+            <select name="ttc-category-options" ng-model="$ctrl.selectedCategory">
+                <option value="">All</option>
+                <option ng-repeat="category in $ctrl.categoriesList" value="{{category}}">{{category}}</option>
```

Comment:
Can these be title-cased so the options are `All`/`Wind`/`Earthquake` with consistent casing?